streams-tests/shared/src/test/scala/zio/stream/experimental/ZTransducerSpec.scala

```diff
@@ -38,7 +38,7 @@ object ZTransducerSpec extends ZIOBaseSpec {
         },
         testM("empty list") {
           val sink = ZTransducer.collectAllN[Int](0)
-          assertM(sink.push.use(_(None)))(equalTo(Chunk.empty))
+          assertM(sink.push.use(_(None)))(equalTo(Chunk(List())))
```

Comment:
Ha great, this is proof `fold` works correctly 😇

Reply:
Exactly!